### PR TITLE
Playlist

### PR DESCRIPTION
**Pull request by vijaytembugade**
- Added Playlist management functionality
- Create a playlist
- Add a video into a playlist

### CODE REVIEW

File: src/Components/PlayListModal/PlaylistModal.js [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+      const response = await createPlayListService(inputPlaylist, token);
+
+      if (response !== undefined && response.status === 201) {
+        dispatch({ type: SET_PLAYLIST, payload: response.data.playlists });
+        toast.success(`${inputPlaylist} playlist created!`);
+      } else {
+        throw new Error("Something went wrong!");
+      }
+    } catch (error) {
+      toast.error(error.message);
```
Comment (horsemaker) [inline src/Components/PlayListModal/PlaylistModal.js:52]: I like this error handling with the help of toast! 👌

File: src/Components/PlayListModal/PlaylistModal.js [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+        token
+      );
+      if (response === undefined) {
+        throw new Error("Something went Wrong!");
+      }
+      setShowPlayListModal(false);
+
+      toast.success(`Video added In ${response.data.playlist.title} Playlist`);
+
+      const playListResponce = await getPlayListDetailsService(token);
```
Comment (horsemaker) [inline src/Components/PlayListModal/PlaylistModal.js]: Typo I guess, `response`

File: src/Components/PlayListModal/PlaylistModal.js [hunk 3]
```diff
@@ -0,0 +1,150 @@
...
+      dispatch({
+        type: SET_PLAYLIST,
+        payload: playListResponce.data.playlists,
+      });
+    } catch (error) {
+      toast.error(error.message);
+    }
+  }
+
+  const videoIncludedPlayLists = playList.reduce((acc, curr) => {
```
Comment (horsemaker) [inline src/Components/PlayListModal/PlaylistModal.js:96]: Reduce! 👌

Comment (horsemaker) [reply src/Components/PlayListModal/PlaylistModal.js:96]: I guess we can also use map here.

**Review by horsemaker — APPROVED** [review]

**Review by preethasri — APPROVED** [review]
nice work . good to merge

**Pull request merged**